examples/hc.conf

```diff
@@ -8,7 +8,7 @@ host3
 # Do a sanity check before proceding
 [script1]
 action=execute
-file=/usr/share/gdeploy/scripts/grafton-sanity-check.sh -d disk1,disk2 -h host1,host2
+file=/usr/share/ansible/gdeploy/scripts/grafton-sanity-check.sh -d disk1,disk2 -h host1,host2
```

Comment:
This is going to be tricky with oVirt releases. Currently I have added path usr/share/ansible/gdeploy/scripts/grafton-sanity-check.sh to my generated configuration. If you change this in future then we also have to change and release a new version. But ovirt release timeline may not align with gdeploy release time line.